gnovm/tests/files/map35.gno

```diff
@@ -0,0 +1,37 @@
+package main
+
+import "strconv"
+
+type stringMap map[*int]int
```

Comment:
maybe a test with a struct as key to test the copy ?

Reply:
the blocker is #4304, so made it simple here.